CHANGELOG.md

```diff
@@ -18,7 +18,9 @@ Our versioning strategy is as follows:
   - Introduced get taxonomy/taxonomies functionality on content client ([#99](https://github.com/Sitecore/content-sdk/pull/99)):
     - Introduced `getTaxonomy` and `getTaxonomies` methods on Content Client class
     - Support pagination for the above methods by implementing an internal `fetchNext` method to handle this in each method
-  - Introduced get locales functionality on content client ([#74](https://github.com/Sitecore/content-sdk/pull/74))([#78](https://github.com/Sitecore/content-sdk/pull/78))   
+  - Introduced `getLocale/getLocales` functionality on content client: 
+    - Updated `getLocale/getLocales` based on new GQL Schemas ([#105](https://github.com/Sitecore/content-sdk/pull/105))   
+    - Initial implementation of `getLocale/getLocales` ([#74](https://github.com/Sitecore/content-sdk/pull/74))([#78](https://github.com/Sitecore/content-sdk/pull/78))
```

Comment:
Let's add this PR reference to "initial implementation", because we need to add an essential information in Changelog, while API changes is not something interesting for customers, since this is feature is totally new
```suggestion
    - Initial implementation of `getLocale/getLocales` ([#74](https://github.com/Sitecore/content-sdk/pull/74))([#78](https://github.com/Sitecore/content-sdk/pull/78))([#105](https://github.com/Sitecore/content-sdk/pull/105))
```